Bugsnag.NET.Tests/Request/NoticeTests.cs

```diff
@@ -24,6 +25,17 @@ public void HasCorrectSchema()
             Assert.IsTrue(jObject.IsValid(schema));
         }
 
+        [Test]
+        public void StackIsAbsent()
+        {
+            const string InternalErrorMessage = "An internal error has occurred!";
+            Exception ex = new Exception(InternalErrorMessage, new Exception("Debug Info: " + Environment.NewLine + "Some debug information goes here..."));
+            var ev = new Event(ex);
+            var notice = new Notice("apikey", new Notifier(), ev);
+            var json = JsonConvert.SerializeObject(notice);
+            // should reach the end of function without any exception thrown
```

Comment:
The test method result (success or fail) is the same in both cases.
According to the http://www.mikedellanoce.com/2010/12/assertdoesnotthrow-why-does-it-exist.html using Assert.DoesNotThrow hides the details of original exception and makes it harder to get the reason of the error.